app/src/features/surveys/components/GeneralInformationForm.test.tsx

```diff
@@ -46,6 +48,24 @@ describe('General Information Form', () => {
                 return { value: item.number, label: `${item.number} - ${item.type}` };
               }) || []
             }
+            funding_sources={[
+              {
+                pfsId: 1,
+                amount: 100,
+                startDate: '2000-04-09 11:53:53',
+                endDate: '2000-05-10 11:53:53',
+                agencyName: 'agency'
+              }
+            ].map((item) => {
+              return {
+                value: item.pfsId,
+                label: `${item.agencyName} | ${getFormattedAmount(item.amount)} | ${getFormattedDateRangeString(
+                  DATE_FORMAT.ShortMediumDateFormat,
+                  item.startDate,
+                  item.endDate
+                )}`
+              };
+            })}
```

Comment:
Not a huge concern, but any reason for doing this mapping here, rather than just supplying the final array of key/values?

Reply:
mainly just so that the endpoint is more generic and doesnt have to return a weird string with a super fe specific formatted string